setup.py

```diff
@@ -41,6 +41,7 @@
     packages=find_packages(exclude=['tests*', 'benchmarks*']),
     include_package_data=True,
     install_requires=[
+        'avro',
```

Comment:
That's false though, it [imports vanilla avro right here](https://github.com/Yelp/data_pipeline/blob/b0a6609188146b9da7d415384047359ceacfb218/data_pipeline/envelope.py#L22-L23)